react_components/PaginationBoxView.js

```diff
@@ -256,6 +275,7 @@ export default class PaginationBoxView extends Component {
         key={index}
         pageSelectedHandler={this.handlePageSelected.bind(null, index)}
         selected={selected === index}
+        rel={this.handlePageRel(index)}
```

Comment:
This may be more pertinently named `getElementPageRel` then _handle_, because we generate a value and do not handle any event here.